### PR TITLE
Jenkins move raspi and bebop builds to regular armhf toolchain

### PR DESCRIPTION
**Pull request by dagar**

### CODE REVIEW

**Comment by dagar** [comment]
It seems to work. http://ci.px4.io:8080/blue/organizations/jenkins/PX4_misc%2FFirmware-hardware_linux/detail/pr-jenkins_raspi/2/pipeline/

image

image

File: Tools/docker_run.sh [hunk 1]
```diff
@@ -5,14 +5,11 @@ if [ -z ${PX4_DOCKER_REPO+x} ]; then
...
 		PX4_DOCKER_REPO="px4io/px4-dev-nuttx:2019-10-24"
-	elif [[ $@ =~ .*navio2.* ]] || [[ $@ =~ .*raspberry.* ]] || [[ $@ =~ .*bebop.* ]]; then
-		# posix_rpi_cross, posix_bebop_default
-		PX4_DOCKER_REPO="px4io/px4-dev-raspi:2019-10-24"
 	elif [[ $@ =~ .*eagle.* ]] || [[ $@ =~ .*excelsior.* ]]; then
 		# eagle, excelsior
 		PX4_DOCKER_REPO="lorenzmeier/px4-dev-snapdragon:2018-09-12"
-	elif [[ $@ =~ .*ocpoc.* ]]; then
-		# aerotennaocpoc_ubuntu
+	elif [[ $@ =~ .*ocpoc.* ]] || [[ $@ =~ .*navio2.* ]] || [[ $@ =~ .*raspberry.* ]] || [[ $@ =~ .*bebop.* ]]; then
```
Comment (mhkabir) [inline Tools/docker_run.sh]: Bebop won't work with the new toolchain but it's probably broken anyway at this point.

Comment (dagar) [reply Tools/docker_run.sh]: It was likely going to be dropped with DriverFramework anyway. Do you know why it wouldn't work?

Comment (mhkabir) [reply Tools/docker_run.sh]: https://github.com/PX4/Firmware/issues/9807

**Pull request merged**